tests/nightly/test_dropout.py

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import sys
+import os
+curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'python', 'unittest'))
+from common import with_seed
+import unittest
+
+import mxnet as mx
+import numpy as np
+from mxnet.test_utils import assert_almost_equal
+from nose.tools import assert_raises
+
+@with_seed()
```

Comment:
would you mind adding a unit test for the RNN op, too? Thx!

Reply:
Added, but cpu does not work, gpu works after my PR. I think it's unrelated to my change, created a separate issue here: https://github.com/apache/incubator-mxnet/issues/16604

Workaround is manual append a dropout layer after rnn on cpu.